client/connection/openvpn.go

```diff
@@ -38,19 +37,12 @@ func ConfigureVpnClientFactory(
 
 		statsSaver := bytescount.NewSessionStatsSaver(statsKeeper)
 
-		consumerCountry, err := locationDetector.DetectCountry()
```

Comment:
Why not detecting on connection start?

Reply:
fixed

Reply:
Does it mean that we are always getting startup location?

Reply:
Getting location on app start